config.mk

```diff
@@ -5,7 +5,7 @@ LD = $(shell which ld)
 ## Common flags
 FLG    = -std=gnu11
 SFLAGS = -O3 -fPIC -D_LARGEFILE64_SOURCE=1 -DHAVE_HIDDEN
-ZLIB   = -DZLIB_API
```

Comment:
This patch is a good opportunity to remove the ZLIB_API guard on test/test_multithread_stress.c.

Reply:
Right, I have to deal with this. If I just change it to always use the nx_ symbols, this test would no longer cover the interface symbols that libnxz provides, and this test doesn't compare the outputs of the regular zlib with libnxz.

I can try to find a way to use the new form of the ZLIB_API makefile variable to conditionally compile this test, like it is done now.